index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+   <nav class="top-header">
+     <a class="welcome identical-to" href="#">Welcome</a>
+      <span class="identical-to">&#8801;</span>
+  </nav>
+  </header> 
+  <main>
+    <section>
+    <div class="intro">
+    <h1 class="first-name">Hey there. I'm Stephen</h1>
+    <h1 class="first-name software-dev">I'm a software developer</h1>
```

Comment:
- [ ] You should use only one h1 tag on the page. Please change the h1 tag in the 'ı am software developer' to hx (h2-h6)tag.
- I know the first reviewer reviewed it like that. But this is misunderstood. You should have only the h1 tag on the page.
![image](https://user-images.githubusercontent.com/38635158/164891856-bda3d178-3bb6-4cd2-b30c-47f5ae4bc6ec.png)

Reply:
good one, thanks